### PR TITLE
[JENKINS-73007] Create-item button no longer disabled when duplicate name present

### PR DESCRIPTION
**Pull request by janfaracik**
See [JENKINS-73007](https://issues.jenkins.io/browse/JENKINS-73007).

Missed this selector in https://github.com/jenkinsci/jenkins/pull/9111 - this PR fixes that so the submit button disables as expected. Tried to reproduce 73007 in an older version of Jenkins and couldn't sadly, the button was enabled despite the duplicate name. To fix this I've added `enableSubmit(false);` to set the button to be disabled when there's a duplicate item.

### Testing done

* Button disables as expected, also when there's a duplicate project name
* Button enables as expected

### Proposed changelog entries

- [JENKINS-73007] Create-item button no longer disabled when duplicate name present

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@jenkinsci/sig-ux 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Review by timja — APPROVED** [review]
untested looks good though

**Review by NotMyFault — APPROVED** [review]
Thanks for the prompt fix!

---

/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Comment by daniel-beck** [comment]
> Create-item button no longer disabled when duplicate name present

I tried with 2.454 and this doesn't do what it claims it does:

> Screenshot 2024-04-18 at 09 36 34

So what does https://github.com/jenkinsci/jenkins/pull/9167/files#diff-4ad0bc0753ae824947f6eca3b6f346433efd8e1bca7c330f894a9b4a4ed4434dR63 actually do? @NotMyFault @timja ?

**Pull request merged**
